cli/main.go

```diff
@@ -17,11 +17,19 @@ Support: support@dispatch.run
 `
 )
 
+var mainCommandText string
+
 func createMainCommand() *cobra.Command {
+	if isDocsBuild {
+		mainCommandText = "This is the main command for Dispatch CLI. Add a subcommand to make it useful."
+	} else {
+		mainCommandText = DispatchCmdLong
+	}
```

Comment:
Instead of this check, could we declare `DispatchCmdLong` in `cli/generate_docs.go` and `generate_docs_default.go` with different values?